### DISCUSSION TITLE
When will be the next release?

### DISCUSSION THREAD

**Discussion by Keyruu** [smallrye/smallrye-graphql#1871] [General]
I need the new GraphQL UI as soon as possible! :D

**Comment by phillip-kruger** [comment]
@jmartisk do we have a release planned ? @Keyruu what is it in the new UI that you need ?

**Reply by Keyruu** [reply]
It looks way better and the refresh button of the schema is nice to have. I think they are also a few UI improvements in terms of searching and displaying schema.

**Reply by jmartisk** [reply]
It is planned to be included in 2.3.0. And that we plan to release just before Quarkus 3.3 to have enough time to integrate it properly.
